server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCicsUnlock.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2024 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.usecases;
+
+import org.eclipse.lsp.cobol.usecases.common.CICSTestUtils;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test CICS UNLOCK command. Documentation link: <a
+ * href="https://www.ibm.com/docs/en/cics-ts/6.x?topic=summary-unlock">UNLOCK
+ * Command</a>
+ *
+ * <p>This class tests all variations of the UNLOCK command found in the link above.
+ */
+public class TestCicsUnlock {
+  private static final String ALL_OPTIONS =
```

Comment:
I suggest switching to variables as operands instead of numbers:
```
 LINE           SOURCE LISTING                                                                                      
                                                                                                                    
00001                     EXEC CICS UNLOCK FILE(12) TOKEN(12) SYSID(31)            00079406                         
00002                                                                              00079304                         
00003                     END-EXEC                                                 00079506                         
00004                                                                              00079604                         
CICS 7.3.0 COMMAND LANGUAGE TRANSLATOR                                            TIME 04.15 DATE 27 NOV 24   PAGE 2
TRANSLATOR DIAGNOSTIC MESSAGES OF SEVERITY W AND ABOVE.                                                             
MESSAGE NUMBER  SEVERITY  LINE NUMBER     MESSAGE                                                                   
DFH7036I           W        00002         PROGRAM DOES NOT BEGIN WITH AN IDENTIFICATION DIVISION.                   
DFH7215I           W        00002         SUB-OPERAND '12' OF 'FILE' KEYWORD SHOULD BE DATATYPE CHARACTER.          
DFH7226I           E        00002         SUB-OPERAND '12' OF 'TOKEN' KEYWORD SHOULD BE A DATA REFERENCE NOT AN     
                                          EXPRESSION OR A CONSTANT. COMMAND NOT TRANSLATED.                         
DFH7215I           W        00002         SUB-OPERAND '31' OF 'SYSID' KEYWORD SHOULD BE DATATYPE CHARACTER.         
END OF TRANSLATOR DIAGNOSTIC MESSAGES.                                                                              
TRANSLATION TIME:-   0.00 MINS.                                                                                     
```